src/index.js

```diff
@@ -52,6 +52,10 @@ function buildBranch(options, callback) {
           rimraf.sync(path.join(options.cwd, file));
         }
       });
+      // Check if build folder exists and is not empty
+      if(!fs.existsSync(options.folder) || !fs.readdirSync(options.folder).length) {
+        callback('Build folder doesn\'t exist or is empty.'); return;
```

Comment:
Hi Mathieu!

thanks for improving this project.

Using the error constructor like this will give a stack to your error which can be useful.

```js
callback(new Error('Build folder doesn\'t exist or is empty.'));
```